src-tauri/Cargo.toml

```diff
@@ -26,8 +26,8 @@ tauri-plugin-shell = "2.0.0"
 tauri-plugin-window-state = "2.0.0"
 tauri-plugin-log = "2.0.0"
 
-jj-lib = { version = "0.27.0", features = ["vendored-openssl", "watchman"] }
-jj-cli = { version = "0.27.0", default-features = false, features = [
+jj-lib = { version = "0.28.1", features = ["vendored-openssl", "watchman"] }
+jj-cli = { version = "0.28.1", default-features = false, features = [
```

Comment:
There were some issues with publishing in `0.28.0`, that's why I needed to go to `0.28.1` directly.

Reply:
There's now a 0.28.2 that you should use instead. Also make sure you update `Cargo.lock` to use `gix-features` 0.41.1.

Reply:
Thanks for the hint! Updated